shared-context/package.json

```diff
@@ -7,6 +7,7 @@
     "clean": "lerna run --scope @shared-context/* --parallel clean"
   },
   "devDependencies": {
-    "lerna": "3.22.1"
+    "lerna": "3.22.1",
+    "webpack-cli": "^5.0.1"
```

Comment:
Change webpack-cli version to 4.10.0 or delete package.json file from changed file at all